Nos/Service/Relay/RelaySubscriptionManager.swift

```diff
@@ -52,9 +54,15 @@ actor RelaySubscriptionManagerActor: RelaySubscriptionManager {
     func all() async -> [RelaySubscription] {
         all
     }
-
-    func sockets() async -> [WebSocket] {
-        sockets
+    
+    // MARK: - Init
```

Comment:
What does `Init` mean here? I was expecting to see an initializer and found none.

Reply:
Oops, I had an `init` here at one point but removed it and forgot to remove the `MARK`.